modules/crypt.cpp

```diff
@@ -501,25 +504,36 @@ class CCryptMod : public CModule {
         }
     }
 
+    void OnNickPrefixCommand(const CString& sCommand) {
+        CString sPrefix = sCommand.Token(1, true);
```

Comment:
Hm, thought space could be used but seems they can't, messages from remote don't arrive at the client.